funnel/assets/js/schedule_view.js

```diff
@@ -1,280 +1,287 @@
-        $('meta[property="og:description"]').attr('content', pageDetails.description);
-        $('link[rel=canonical]').attr('href', pageDetails.url);
-        $('meta[property="og:url"]').attr('content', pageDetails.url);
-      },
-      handleBrowserHistory() {
-        // On closing modal, update browser history
-        $("#session-modal").on($.modal.CLOSE, () => {
-          this.set('modalHtml', '');
-          window.history.pushState('', '', this.get('pageDetails')['url']);
-          this.updateMetaTags(this.get('pageDetails'));
-        });
-        // Event listener for back key press since opening modal update browser history
-        $(window).on('popstate', () => {
-          if(this.get('modalHtml')) {
-            $.modal.close();
-          } else if(window.history.state) {
-            // Open the modal with previous session viewed
-            this.openModal(window.history.state.html, window.history.state.backPage, window.history.state.pageDetails);
-          }
-        });
-      },
-      openModal: function(sessionHtml, backPage, pageDetails) {
-        this.set('modalHtml', sessionHtml);
-        $("#session-modal").modal('show');
-        window.history.pushState({html: sessionHtml, backpage: backPage, pageDetails: pageDetails}, '', backPage);
-        this.updateMetaTags(pageDetails);
-      },
-      showSessionModal: function(event, activeSession) {
-        let sessionModalUrl, sessionUuid, backPage, pageDetails;
-        sessionModalUrl = event ? this.get(event.keypath + '.talks.modal_url') : activeSession.modal_url;
-        sessionUuid = event ? this.get(event.keypath + '.talks.url_name_suuid') : activeSession.url_name_suuid;
-        backPage = this.get('pageDetails')['url'] + '/' + sessionUuid;
-        if (event) {
-          pageDetails = {
-            title: this.get(event.keypath + '.talks.title') + ' — ' + this.get('pageDetails')['projectTitle'],
-            pageTitle: this.get(event.keypath + '.talks.title'),
-            description: this.get(event.keypath + '.talks.speaker') ? this.get(event.keypath + '.talks.title') + ' by ' + this.get(event.keypath + '.talks.speaker') : this.get(event.keypath + '.talks.title') + ", " + this.get('pageDetails')['projectTitle'],
-            url: backPage
-          };
-        } else {
-          pageDetails = {
-            title: activeSession.title + ' – ' + this.get('pageDetails')['projectTitle'],
+          return descriptionHtml.replace(/<img[^>]*>/g, '').substring(0, 400).concat('..');
+        },
+        updateMetaTags(pageDetails) {
+          $('title').html(pageDetails.title);
+          $('meta[name="DC.title"]').attr('content', pageDetails.pageTitle);
+          $('meta[property="og:title"]').attr('content', pageDetails.pageTitle);
+          $('meta[name=description]').attr('content', pageDetails.description);
+          $('meta[property="og:description"]').attr('content', pageDetails.description);
```

Comment:
If the `attr` function escapes HTML, then we don't need the Jinja2 `e` filter when the data is loaded.